homeassistant/components/huawei_lte/__init__.py

```diff
@@ -339,6 +342,12 @@ def get_connection() -> Connection:
 
     try:
         connection = await hass.async_add_executor_job(get_connection)
+    except (
+        LoginErrorUsernameWrongException,
+        LoginErrorPasswordWrongException,
+        LoginErrorUsernamePasswordWrongException,
+    ) as ex:
```

Comment:
Consolidating the errors to have a single base class (like `InvalidLoginException` or similar) would simplify the code, especially when they are handled the same, if the upstream is perceptive for such changes.

Reply:
Good idea, let's see: https://github.com/Salamek/huawei-lte-api/pull/146

I suggest we don't wait for that though, because it would balloon the PR scope a bit by requiring a dependency upgrade.

Reply:
Upstream was quick, upgraded to 1.6.3 and addressed in 733d96e041bc9efcfbba6639620f19d97bc6bd6e